src/rules/require-baseline.js

```diff
@@ -556,6 +556,11 @@ export default {
 				node,
 			) {
 				for (const child of node.children) {
+					// ignore unknown media conditions - no-invalid-at-rules already catches this
```

Comment:
Can you also a add a test that has an invalid media condition to verify that it's ignored?